### PR TITLE
<ModalMobileLayout/> - fix Title takes 100% width even when close button appears

### PR DESCRIPTION
**Pull request by roibWix**
### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  Project Name: **wix-ui-tpa**

  You can find it under one of the following URLs:
  * https://bo.wix.com/pages/wix-ui-tpa.pr_6876/

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  Project Name: **wix-style-react**

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6876

File: packages/wix-style-react/src/ModalMobileLayout/ModalMobileLayout.st.css [hunk 1]
```diff
@@ -14,13 +14,14 @@
...
 .root {
-    -st-states: fullscreen, stickyTitle, stickyFooter, noTitle, noFooter;
+    -st-states: fullscreen, stickyTitle, stickyFooter, noTitle, noFooter, closeBtn;
     height: 100%;
     width: 100%;
     padding: 30px;
     display: flex;
     align-items: center;
     box-sizing: border-box;
+  outline: 1px solid black;
```
Comment (EgoziE) [inline packages/wix-style-react/src/ModalMobileLayout/ModalMobileLayout.st.css:24]: Doesn't looks like something from our design, can you show me why you added this?

**Review by EgoziE — APPROVED** [review]
Approved, just ask a UX to take a quick look

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

A deploy preview of your PR was successfully released 🖼.

Project Name: **wix-style-react**

You can find it under one of the following URLs:
* https://preview.wix-style-react.com/pr_6876

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

A deploy preview of your PR was successfully released 🖼.

Project Name: **wix-ui-tpa**

You can find it under one of the following URLs:
* https://bo.wix.com/pages/wix-ui-tpa.pr_6876/

**Pull request closed without merging**